### DISCUSSION TITLE
Free bot for Forge of Empires

### DISCUSSION THREAD

**Discussion by carlosroxo1** [ThScEo/AutoForge#76] [Q&A]
Hey! Im new here, so I don't understand very well some things.
The ReadMe file says some options are free, while others are premium. I would like to know how can I use just the free features.
When opening the bot, it asks me for a key. I've used the trail key and the period is over. How can I open the bot without a valid key? That is, just using its free features.

**Comment by ThScEo** [comment]
Free with trial key, once used, 4€/ month

**Reply by carlosroxo1** [reply]
May I ask, then, what the difference is between free features and premium features?

Also, if you don't mind me asking. How were you able to generate signatures for the request's headers? I've been also trying to implement stuff on my own but I can't seem to figure that one out.

**Comment by kpinini** [comment]
I'm having this issue :
Create mysql database empire
CREATE DATABASE empire

ERROR 1007 (HY000) at line 1: Can't create database 'empire'; database exists

Can someone help me?
